src/main/java/net/shadowmage/ancientwarfare/npc/item/ItemFoodBundle.java

```diff
@@ -10,7 +10,7 @@
 
 public class ItemFoodBundle extends ItemFood implements IClientRegister {
 	public ItemFoodBundle() {
-		super(15, 1f, true);
+		super(15, 20.6f, true);
```

Comment:
Saturation modifier usually is from 0 (no saturation) up to 1 (full saturation), 20 sounds like way too much and not even sure how the game works with that.